apps/src/applab/AppLabView.jsx

```diff
@@ -64,19 +63,12 @@ var AppLabView = React.createClass({
         <ProtectedStatefulDiv
             id="codeWorkspace"
             renderContents={this.props.renderCodeWorkspace} />
-      </StudioAppWrapper>
+      </ConnectedStudioAppWrapper>
     );
-  },
-
-  shouldHideToggle: function () {
-    return this.props.isShareView || this.props.isDesignModeHidden;
-  },
-
-  shouldHideViewDataButton: function () {
-    return this.props.isViewDataButtonHidden ||
-        this.props.isDesignModeHidden ||
-        this.props.isShareView ||
-        !this.props.isEditingProject;
   }
 });
-module.exports = AppLabView;
+module.exports = connect(
+  (state) => ({
```

Comment:
:stuck_out_tongue_closed_eyes: Fine.  Thought I might get away with it in JSX...